tsfresh/feature_selection/feature_selector.py

```diff
@@ -102,15 +102,10 @@ def check_fs_sig_bh(X, y,
             "Feature",
             "type" (binary, real or const),
             "p_value" (the significance of this feature as a p-value, lower means more significant)
-            "rejected" (if the Benjamini Hochberg procedure rejected this feature)
+            "relevant" (True if the Benjamini Hochberg procedure rejected the null hypothesis for this feature)
```

Comment:
You could explain the null hypothesis here again for people not reading the rest :-)